applications/dashboard/modules/class.navmodule.php

```diff
@@ -1,349 +1,118 @@
-<?php
+<?php if (!defined('APPLICATION')) exit();
+use NestedCollection;
```

Comment:
It looks like `NestedCollection` is a trait, not a namespace, so the `use` keyword here, outside of a class, probably isn't going to have the intended effect.  `NavModule` already includes the `NestedCollection` trait.